.github/workflows/markdown-link-checker.yml

```diff
@@ -23,5 +23,7 @@ jobs:
       - name: markdown-link-check
         if: steps.verify-changed-files.outputs.files_changed == 'true'
         uses: gaurav-nelson/github-action-markdown-link-check@master
-#        with:
+        with:
+          base-branch: 'main'
```

Comment:
downside to this is that a user may not be using main or may still have master as the primary branch. Trying to keep it generic so anybody can reference and use the job.